src/internal/version/backup.go

```diff
@@ -1,5 +1,7 @@
 package version
 
+// TODO(ashmrtn): If bumping this adjust OneDriveXLocRef. It's not used in prod
+// yet but does gate some tests.
```

Comment:
This makes me a uneasy.  I'd rather changes like this commit to a version and bump.  In the unlikely event that some other version 7 goes to prod while this is still in queue, we can change the PR.  In the similarly-unlikely event that two PRs declare a version bump, labeling them both 7 is benign.

Or- if you don't plan to commit to a version- set the const to `Backup+1`.